patterns/1-initial/sustainable-innersource.md

```diff
@@ -0,0 +1,86 @@
+## Title
+
+Sustainable InnerSource Programme
+
+## Patlet
+
+I've started an InnerSource programme - what next? How do I ensure InnerSource remains a part of how we create software in the long term.
+
+## Problem
+
+Given an InnerSource programme and/or community has been started within your organisation, you are unsure how to scale and/or sustain to ensure that InnerSource remains part of culture and best practice in the long term. When scaling and sustaining an InnerSource programme fails, it leads to problems including unusable projects/products, an unengaged community, and loss of support from leadership.
+
+## Context
+
+* You are worried that key people will move on to new roles, momentum will be lost and silently die, or future leaders will stop supporting InnerSource.
+* You are an individual or organization who is concerned with the fragile state and future of highly-used and impactful InnerSource projects.[1] https://sustainoss.org/assets/pdf/SustainOSS-west-2017-report.pdf
+* The quantity and quality of InnerSource projects at your organization are changing.
+
+## Forces
```

Comment:
In general, forces should indicate a situation where **trade-offs** exist. E.g., the IS program is small, so the reliance on key people to keep it going is high (implying that if you scale up the program, there would be less reliance on key people).

All four of the stated forces appear to be **context** elements as they are currently represented. E.g., the program lacks full time support--this is a condition that gives context to the problem. A related force may be: creating the political will to give full time support to InnerSource (e.g., through an ISPO) requires persuading management to see the explicit value of such an investment, but the IS program is new and data isn't yet available to prove this.

E.g., a clear, easily explainable vision of the future of the IS program would help win management support, but the IS projects that are interested in the program are too few and too varied to tie together in this way. Or maybe there is no one at an IS program level with the bandwidth to create such a vision and champion it to management.

Reply:
Thanks for your help @NewMexicoKid. 

This is another case where authors struggle with Context vs Forces (I often do myself to be honest).
Linking to the issue related to [improving our Context/Forces explanations](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/419) for future authors.

Reply:
Addressed by faecab8ded8264c7d2bd9fbbf70fbf2119d4eecd